# Ardrone3PilotingStateSpeedChanged
# auto-generated from https://raw.githubusercontent.com/Parrot-Developers/libARCommands/7e2f55fafcd45ba2380ca2574a08b7359c005f47/Xml/ARDrone3_commands.xml
# Do not modify this file by hand. Check scripts/meta folder for generator files.
#
# SDK Comment: Drone speed changed

Header header

# Speed on the x axis (when drone moves forward, speed is &gt; 0) (in m/s)
float32 speedX
# Speed on the y axis (when drone moves to right, speed is &gt; 0) (in m/s)
float32 speedY
# Speed on the z axis (when drone moves down, speed is &gt; 0) (in m/s)
float32 speedZ
